core/src/main/java/org/elasticsearch/bootstrap/SystemCallFilter.java

```diff
@@ -272,27 +272,6 @@ private static int linuxImpl() {
                 "with CONFIG_SECCOMP and CONFIG_SECCOMP_FILTER compiled in");
         }
 
-        // pure paranoia:
-
-        // check that unimplemented syscalls actually return ENOSYS
-        // you never know (e.g. https://code.google.com/p/chromium/issues/detail?id=439795)
-        if (linux_syscall(999) >= 0) {
-            throw new UnsupportedOperationException("seccomp unavailable: your kernel is buggy and you should upgrade");
-        }
-
-        switch (Native.getLastError()) {
-            case ENOSYS:
-                break; // ok
-            case EPERM:
```

Comment:
EPERM static constant can be removed if it is not use here anymore

Reply:
I pushed c3f8be92488829ddd173c34d58e85fdac37ff7cc.